kiwi/partitioner/dasd.py

```diff
@@ -72,3 +72,7 @@ def create(self, name, mbsize, type_name, flags=None):
     def set_flag(self, partition_id, flag_name):
         # on an s390 dasd device there are no such flags
         pass
+
+    def set_hybrid_mbr(self):
+        # on an s390 dasd device there is no concept of an hybrid MBR
```

Comment:
In fact, can you set the function as unimplemented in the parent class? This really *only* makes sense for gpt. ever.